services/like.go

```
@@ -410,6 +412,144 @@ func (ls *LikeService) StartUpdateTicker() {
+		return fmt.Errorf("the user %s is already followed user %s", UserID, FolloweeID)
+	}
+	//添加用户到粉丝合集
+	utils.Client.SAdd(utils.Ctx, key, UserID)
+	//更新用户关注数
+	key = "User:followees"
+	utils.Client.HIncrBy(utils.Ctx, key, UserID, 1)
+	Record := models.FollowMessage{
+		FollowerID: UserID,
+		FolloweeID: FolloweeID,
+	}
+	go ls.ld.SyncFollowMessageToDB(&Record)
```

golangci-lint (1.62.2)

Error return value of `ls.ld.SyncFollowMessageToDB` is not checked

(errcheck)